modules/editable-layers/src/editable-layers/editable-layer.ts

```diff
@@ -154,7 +155,13 @@ export abstract class EditableLayer<
       mapCoords,
       pointerDownScreenCoords: screenCoords,
       pointerDownMapCoords: mapCoords,
-      cancelPan: event.stopImmediatePropagation,
+      cancelPan: () => {
+        if (this.props.onCancelPan) {
+          this.props.onCancelPan();
+        }
```

Comment:
```suggestion
        this.props.onCancelPan?.();
```
Or make the default prop an empty function?